migration/migrating_4_1_4/deploying_cam.adoc

```diff
@@ -1,25 +1,66 @@
-[id='deploying-cam_{context}']
-= Deploying the Cluster Application Migration (CAM) tool
+[id='deploying-cam-4-1-4']
+= Deploying the Cluster Application Migration tool
 include::modules/common-attributes.adoc[]
 :context: migrating-4_1-4_x
 :migrating-4_1-4_x:
 
 toc::[]
 
-Deploying the Cluster Application Migration (CAM) tool requires installing the Cluster Application Migration Operator on the xref:../../migration/migrating_4_1_4/deploying_cam.adoc#installing-cam-operator-ocp-4_sourcecluster-4_1-4_x[{product-title} 4.1 source] and xref:../../migration/migrating_4_1_4/deploying_cam.adoc#installing-cam-operator-ocp-4_targetcluster-4_1-4_x[{product-title} {product-version} target] clusters.
+You can install the Cluster Application Migration Operator on your {product-title} {product-version} target cluster and 4.1 source cluster. The Cluster Application Migration Operator installs the Cluster Application Migration (CAM) tool (CAM web console and Migration controller) on the target cluster by default.
 
-The Cluster Application Migration Operator installs the CAM tool (CAM web console and Migration controller) on the target cluster by default. You can link:https://access.redhat.com/articles/5064151[install the CAM tool on {product-title} 3 and remote clusters].
+[NOTE]
+====
+Optional: You can configure the Cluster Application Migration Operator to install the CAM tool link:https://access.redhat.com/articles/5064151[on an {product-title} 3 cluster or on a remote cluster].
```

Comment:
In this case, should this be a 4 cluster, not a 3?

Reply:
Read the link.